pkgs/applications/misc/tandoor-recipes/default.nix

```diff
@@ -133,8 +124,13 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
 
   nativeCheckInputs = with python.pkgs; [
     pytestCheckHook
+    pytest-asyncio
+    pytest-cov
     pytest-django
     pytest-factoryboy
+    pytest-html
+    pytest-xdist
+    mock
```

Comment:
Please sort the list.

I also wonder whether `pytest-xdist` should be used on Hydra.

Reply:
I don't know. I just checked the requirements.txt diff and didn't thought about it.

I'm am currently checking if it builds without

Reply:
We need to apply this patch if we want to get rid of `pytest-xdist` :
```
diff --git a/pytest.ini b/pytest.ini
index a3d26ec4..8bdf12fb 100644
--- a/pytest.ini
+++ b/pytest.ini
@@ -3,5 +3,5 @@ DJANGO_SETTINGS_MODULE = recipes.test_settings
 testpaths = cookbook/tests
 python_files = tests.py test_*.py *_tests.py
 # uncomment to run coverage reports
-addopts = -n auto --cov=. --cov-report=html:docs/reports/coverage --cov-report=xml:docs/reports/coverage/coverage.xml --junitxml=docs/reports/tests/pytest.xml --html=docs/reports/tests/tests.html
+addopts = --cov=. --cov-report=html:docs/reports/coverage --cov-report=xml:docs/reports/coverage/coverage.xml --junitxml=docs/reports/tests/pytest.xml --html=docs/reports/tests/tests.html
 # addopts = -n auto --junitxml=docs/reports/tests/pytest.xml --html=docs/reports/tests/tests.html
\ No newline at end of file
```

I'm not aware of hydra limitations about parallelism. But my guess is that we have those two solutions regarding to how hydra works :
 - The sandbox hides the "unused" cpus so we need to change nothing
 - We keep `pytest-xdist` but change the `-n auto` to `-n $NIX_BUILD_CORES` (expanding the variable this I don't think `pytest.ini` support variable expansion)

Reply:
Turns out we should use the second option according to what I read. I pushed an implementation accordingly